dhall-docs/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+1.0.0
+
+* [Generate documentation from header comment](https://github.com/dhall-lang/dhall-haskell/pull/1929)
+    * `dhall-docs` will now render a module header from a comment at
+      the beginning of a Dhall file
+    * The comment syntax is essentially markdown with the same whitespace
+      and indentation rules as multi-line strings in Dhall
+    * Both block comments and multiple single-line comments are supported
+* [Jump to imports](https://github.com/dhall-lang/dhall-haskell/pull/1959)
+    * The documentation will now generate links for imports found within the
+      rendered source code
```

Comment:
I don't know if we could be more specific about this? In this moment, we only support relative and remote imports
```suggestion
* [Jump to imports](https://github.com/dhall-lang/dhall-haskell/pull/1959)
    * The documentation will now generate links for relative and absolute imports found within the
      rendered source code
```